package.json

```diff
@@ -55,23 +57,30 @@
     "@babel/plugin-transform-parameters": "^7.1.0",
     "@babel/preset-env": "^7.1.0",
     "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.6.0",
     "@babel/register": "^7.0.0",
     "@emotion/babel-preset-css-prop": "10.0.9",
     "@storybook/addon-a11y": "^5.1.9",
     "@storybook/addon-actions": "^5.1.9",
     "@storybook/addon-centered": "^5.1.9",
+    "@storybook/addon-info": "^5.2.5",
     "@storybook/addon-knobs": "^5.1.9",
     "@storybook/addon-links": "^5.1.9",
     "@storybook/addon-storysource": "^5.1.9",
     "@storybook/addon-viewport": "^5.1.9",
     "@storybook/react": "~5.1.9",
     "@svgr/rollup": "^4.0.2",
     "@svgr/webpack": "^4.0.2",
+    "@types/enzyme": "^3.10.3",
+    "@types/jest": "^24.0.20",
+    "@types/react-transition-group": "^4.2.3",
+    "@types/tinycolor2": "^1.4.2",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
     "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.4",
     "babel-plugin-transform-svg-import-to-string": "^0.0.1",
+    "babel-preset-react-app": "^9.1.0",
```

Comment:
Based on the previous comments, we should remove this too.

Reply:
done